tasks/closerfunction.js

```diff
@@ -0,0 +1,15 @@
+// - Closer task
+// Create a function increaseCount which create another function which increase the value by one
+
+function increaseCount() {
+    // some code
+    var increase = 1;
```

Comment:
it's a variable, need call variable depending on their entity 
if this count please call a `counter` or a `result` 
What's the `increase ` ??? who increase? where is increase? and so on